src/AutoAdmin.tsx

```diff
@@ -238,7 +238,7 @@ const attributeToInput = (input: AutoAdminAttribute) => {
       return <BooleanInput label={input.label} source={input.attribute} options={input.fieldOptions} />;
     case Date:
       return input.fieldOptions && input.fieldOptions.showTime ? (
-        <DateTimeInput label={input.label} source={input.attribute} options={input.fieldOptions} />
+        <DateTimeInput label={input.label} source={input.attribute} options={{ ampm: false }} />
```

Comment:
@golastra Maybe pass the rest of options so we can pass custom options in the future
```suggestion
        <DateTimeInput label={input.label} source={input.attribute} options={{ ...input.fieldOptions , ampm: false }} />
```